src/main/java/com/puppycrawl/tools/checkstyle/JavadocPropertiesGenerator.java

```diff
@@ -103,36 +101,42 @@ public static void main(String... args)
      * @param inputFile .java file
      * @param outputFile .properties file
      * @throws CheckstyleException if a javadoc comment can not be parsed
-     * @throws IOException if there is a problem with files access
+     * @noinspection IOResourceOpenedButNotSafelyClosed
```

Comment:
Please explain.

Reply:
`new FileOutputStream(outputFile)` may throw a `FileNotFoundException` and `PrintWriter::println` may throw an `IOException`.
So there must be two nested try blocks or FileOutputStream may be not safely closed.
Both cases are prohibited by default, so one of this inspections must be muted.

Actually, the right solution is to use try-with-resource block, but this is not possible for now due to Cobertura limitations.
In the future, when the Cobertura will be replaced, we should review every `finally` block.

Reply:
NO suppression is allowed, please update IDEA inspections, as I suggested.